src/flow.h

```diff
@@ -117,6 +117,9 @@ typedef struct AppLayerParserState_ AppLayerParserState;
 #define FLOW_TS_APP_UPDATED BIT_U32(29)
 #define FLOW_TC_APP_UPDATED BIT_U32(30)
 
+#define FLOW_TS_APP_UPDATE_NEXT BIT_U32(31)
```

Comment:
Could these get a comment ? Not sure I understand the purpose

Reply:
yes